contracts/axelarnet-gateway/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "axelarnet-gateway"
+version = "0.1.0"
+rust-version = { workspace = true }
+edition = "2021"
```

Comment:
is this the right year?

Reply:
lol, afaik Rust 2024 edition hasn't been released yet